package.json

```
@@ -91,6 +91,59 @@
             }
           }
         }
+      },
+      {
+        "name": "apply_projectTemplate",
+        "when": "confluent.chatParticipantEnabled",
+        "tags": [
+          "projects"
+        ],
+        "displayName": "Apply Project Template",
+        "modelDescription": "Apply a selected project template to generate a streaming data project or application. Use this when the user selects a template and provides the necessary options.",
+        "canBeReferencedInPrompt": false,
+        "icon": "$(checklist)",
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "The name of the template to apply."
+            },
+            "options": {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "description": "Key-value pairs of options to configure the template."
```

Not necessary for this branch, but some hinting may be helpful here if the models have problems looking up options:
```suggestion
              "description": "Key-value pairs to fill a project template, based on 'get_projectOptions'."
```